openapi.yaml

```diff
@@ -3645,30 +3651,35 @@ components:
                     description: A bool representing whether or not to require MFA for requesting access to this group.
                     example: False
                     type: boolean
+                    x-speakeasy-ignore: true
                 auto_approval:
                     description: A bool representing whether or not to automatically approve requests to this group.
                     example: False
                     type: boolean
+                    x-speakeasy-ignore: true
                 request_template_id:
                     description: The ID of the associated request template.
                     example: 06851574-e50d-40ca-8c78-f72ae6ab4304
                     format: uuid
                     type: string
+                    x-speakeasy-ignore: true
```

Comment:
nixed because `request_template_id` should come from the request_configuration. top level field is deprecated